lib/config.js

```diff
@@ -131,6 +131,7 @@ config.shibLinkColors = {
 config.hasAzure = false;
 config.hasOauth = false;
 config.syncExamIdAccessRules = false;
+config.ptHost = 'http://localhost';
```

Comment:
```suggestion
config.ptHost = 'http://localhost:4000';
```
If this is supposed to be the default for local environment tests, this is more likely to be the correct option.